TestsZipCodeEndpoint.cs

```diff
@@ -0,0 +1,195 @@
+﻿using Newtonsoft.Json;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using RestSharp;
+
+namespace APIAutomation.Tests
+{
+    [TestFixture]
+    public class ZipCodeEndpointGETTests
+    {
+        private RestClient _client;
+        private RestRequest _request;
+
+        [SetUp]
+        public void Setup()
+        {
+            var client = ClientForReadScope.GetInstance("http://localhost:8000", "0oa157tvtugfFXEhU4x7", "X7eBCXqlFC7x-mjxG5H91IRv_Bqe1oq7ZwXNA8aq");
+            _client = client.GetRestClient();
+            _request = new RestRequest("/zip-codes", Method.Get);
+        }
+
+        [Test]
+        public void GetAllZipCodes_ReturnsAllAvailableZipCodes()
+        {
+
+            _client.AddDefaultHeader("Accept", "application/json");
+            RestResponse response = _client.Execute(_request);
+
+            List<string> expectedZipCodes = new List<string> { "12345", "23456", "ABCDE" };
+            List<string> actualZipCodes = JsonConvert.DeserializeObject<List<string>>(response.Content);
+
+            Assert.Multiple(() =>
+            {
+                //The test fails, Status Code for successful Get Method equals 201
+                Assert.That((int)response.StatusCode, Is.EqualTo(200));
```

Comment:
You may check status code like that - `Assert.AreEqual(HttpStatusCode.OK, userResponse.StatusCode)`

Reply:
I've replaced with 'Assert.That(response.Content, Does.Contain("OK"));', because for 'Assert.AreEqual(HttpStatusCode.OK, userResponse.StatusCode)' I receive: ''Assert' does not contain a definition for 'AreEqual''